packages/tooltips/src/containers/TooltipContainer.js

```diff
@@ -90,13 +92,18 @@ class TooltipContainer extends ControlledComponent {
      * @param {Function} renderProps.getTriggerProps - Props to be spread onto the trigger element
      * @param {Function} renderProps.isVisible - Whether the Tooltip is currently visible
      */
-    trigger: PropTypes.func
+    trigger: PropTypes.func,
+    /**
+     * The z-index of the popper.js placement container
+     */
+    zIndex: PropTypes.number
   };
 
   static defaultProps = {
     placement: 'top',
     eventsEnabled: true,
-    delayMilliseconds: 500
+    delayMilliseconds: 500,
+    zIndex: 1000
```

Comment:
Unlike the menu component (and so that this is not a breaking change) the default should be `'auto'` – which is the CSS default. Note that the PropType needs to be flexible.

Reply:
Or, perhaps, no default is best. Either way, the PropType must accept valid CSS values (auto, inherit, etc)